src/app/components/Metadata/propTypesCheck.js

```diff
@@ -0,0 +1,17 @@
+import { checkPropTypes } from 'prop-types';
+
+const propTypeCheck = (props, componentName, propType) => {
```

Comment:
Isn't the second argument here the prop name? The component name is hardcoded to 'Metadata' on line 11

Reply:
good point, will fix 🤦‍♀